### PR TITLE
IA-1366 Enable welder in /etc/jupyter/nbconfig/notebook.json

### PR DESCRIPTION
**Pull request by rtitle**
JIRA: https://broadworkbench.atlassian.net/browse/IA-1366

Found during bug bash.

Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: http/src/main/resources/jupyter/startup.sh [hunk 1]
```diff
@@ -28,8 +28,9 @@ if [ "$DEPLOY_WELDER" == "true" ] ; then
     gcloud auth configure-docker
     docker-compose -f /etc/welder-docker-compose.yaml up -d
 
-    # Move existing notebooks to new notebooks dir
```
Comment (rtitle) [inline http/src/main/resources/jupyter/startup.sh:31]: I decided to remove this `mv` command:
1. It's scary to `mv` users' files
2. Terra UI re-localizes everything anyway
3. The directory it `mv`s to isn't even correct (it doesn't use the `edit` or `safe` leaf paths)

I will re-test with this change.

File: http/src/main/resources/jupyter/startup.sh [hunk 2]
```diff
@@ -28,8 +28,9 @@ if [ "$DEPLOY_WELDER" == "true" ] ; then
...
-    docker exec -i $JUPYTER_SERVER_NAME bash -c "ls -I jupyter.log -I localization.log -I notebooks /home/jupyter-user | xargs -d '\n'  -I file mv file $NOTEBOOKS_DIR"
+    # Enable welder in /etc/jupyter/nbconfig/notebook.json (which powers the front-end extensions like edit.js and safe.js)
+    docker exec -u root -i $JUPYTER_SERVER_NAME bash -c \
```
Comment (rtitle) [inline http/src/main/resources/jupyter/startup.sh:35]: `jq` for the win

**Review by Qi77Qi — APPROVED** [review]

**Comment by rtitle** [comment]
jenkins retest

**Comment by rtitle** [comment]
fiab-start failed, looks transient.

jenkins retest

**Pull request merged**